GSoC-Ideas.md

```diff
@@ -38,4 +38,28 @@ The aims of the project are as follows:
 * _Mentors:_ Sean Goggins, Matt Germonprez
 
 
-## Idea: TBD
+## Idea: Implement the Social Currencyy Metrics System in Grimoire Labs
+
+[ Micro-tasks and place for questions ](https://github.com/chaoss/augur/issues/545)
+
+The Social Currency Metrics System (SCMS) is a qualitative data collection, processing, and measurement system that augments quantitative community management metrics already available in CHAOSS properties. Implementing the SCMS will ultimately help community leaders, power users, and other stakeholders leverage qualitative data for social listening so that they can rely less on the behaviors quantitative data tracks and more on community sentiment. The SCMS empowers community leaders to make decisions based on what community members freely share about their opinions, wants, and needs.
+
+The SCMS shows why trends occur and identifies commonly missed pitfalls in conclusions taken from quantitative data. With an SCMS platform built natively into a CHAOSS Bitergia property, open-source communities can use it to facilitate members’ input in decisions essential to community health.
```

Comment:
-- Should this be GrimoireLab (not Bitergia)? If not, the line should be modified to not include Bitergia as the programs are about helping open source projects (GL), not companies.